website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1058,10 +1058,10 @@ $ curl \
 ```
 
 <Note title="Important change to supported versions">
-   As of 1.16.7, 1.17.3 and later, 
-   the <a href="/vault/docs/concepts/billing-start-date">billing start date</a> automatically 
+   As of 1.16.7, 1.17.3 and later,
+   the <a href="/vault/docs/concepts/billing-start-date">billing start date</a> automatically
```

Comment:
wondering if the space that got deleted after "automatically" will delete the space in the website too. If that happens, we will have to add a space back here.

Reply:
Confirmed that the space didn't get delete on the website!